src/main/java/stream/flarebot/flarebot/util/MessageUtils.java

```diff
@@ -122,9 +127,22 @@ public static void editMessage(Message message, String content) {
     }
 
     public static EmbedBuilder getEmbed() {
-        return new EmbedBuilder()
-                .setAuthor("FlareBot", "https://flarebot.stream", flareBot.getSelfUser()
-                        .getEffectiveAvatarUrl());
+        if (cachedJDA == null || cachedJDA.getStatus() != JDA.Status.CONNECTED)
+            cachedJDA = flareBot.getClient();
```

Comment:
If this is planned to be used often I would make a private method for this purpose

Reply:
We could, I just did it here since we call getEmbed a good 100k+ times in a day or two.